apps/explorer/src/app/components/blocks/home/blocks-refetch.tsx

```diff
@@ -0,0 +1,11 @@
+interface BlocksRefetchProps {
+  refetch: () => void;
+}
+
+export const BlocksRefetch = ({ refetch }: BlocksRefetchProps) => {
+  return (
+    <button onClick={() => refetch()} className="underline mb-28">
```

Comment:
I can see this being a <ButtonLink /> component in UI toolkit. Basically something we semantically need to be a button but looks like a text link

Reply:
Good shout. I've made a ticket for this: https://github.com/vegaprotocol/frontend-monorepo/issues/75